src/app/leverage/utils/fetch-leverage-token-prices.ts

```diff
@@ -55,8 +55,7 @@ export async function fetchLeverageTokenPrices(
 
     const enrichedTokens = tokenBalances.map((token, idx) => {
       const usd =
-        parseFloat(formatAmountFromWei(token.balance, token.decimals, 3)) *
-        tokenPrices[idx]
+        parseFloat(formatWei(token.balance, token.decimals)) * tokenPrices[idx]
```

Comment:
I guess you checked this already but just making sure this was having only 3 digits after the comma before but I guess it's formatted anyway later in the flow?

Reply:
Didn't see anything in testng

Reply:
Yep - formatted a few lines below here (and not used elsewhere): `size: formatPrice(usd),`